### ISSUE TITLE
[Vanilla Kubernetes] Grafana is not working on K8S Azure cluster.

### ISSUE THREAD

**Issue by theobolo** [fabric8io/fabric8#5687]
Since the last version of Fabric8 doesn't show the "Run..." button, i've tried to install the management template by modifying the .yml and config it to be a "Kind=List" and not a Template.

So i can deploy the management suite but on my config, neither grafana or kibana are working. The services are looking for a bad URI if you access it by the classic api URL

ex : http://"your-domain":8080/api/v1/proxy/namespaces/default/services/grafana

Grafana is looking for

**http://"your-domain":8080/api/dashboards/home**

and not

**http://"your-domain":8080/api/v1/proxy/namespaces/default/services/grafana/api/dashboards/home**

witch is working.

![image](https://cloud.githubusercontent.com/assets/7281773/12747561/0743af32-c9a5-11e5-94d9-36510dfa114b.png)

Otherwise i tried to map the container port to the outside by exposing it with and Azure endpoint

![image](https://cloud.githubusercontent.com/assets/7281773/12747613/5471bb46-c9a5-11e5-8051-5017e5af1efd.png)

Grafana is looking for

http://"your-domain":35642/api/datasources/**proxy/1/api/v1/**query_range?query=etc........

Grafana pod logs :

![image](https://cloud.githubusercontent.com/assets/7281773/12747620/5d56cfa8-c9a5-11e5-82f0-99c079e22e16.png)

Any ideas ?

**Comment by theobolo** [comment]
It seems that all the management modules, Grafana, Prometeus and Kibana are not working on the Azure "getting started guide" cluster, except Gogs.

Also the CD pipeline is not working at all.

**Comment by rawlingsj** [comment]
@theobolo at the moment I expect the CD pipeline will have issues as it uses the openshift build config resource when creating projects / apps.  We need to do a bit of development now that the `thirdpartyresource` is available in the newer version of Kubernetes.  Apologies, we should maybe make that clear in the console.

**Comment by theobolo** [comment]
@rawlingsj Ok James so for the moment there is no "official" support if i want to install all the Fabric8 stuff on an Azure cluster for example ?

I've wanted to use K8S and Fabric8 with the CD pipeline to provide a full scalable environnement for my company, but it seems that is a little bit too early for that, am i right ?

Thanks for you answer.

ps: If i try to run Openshift Origin inside my Kubernetes cluster it will be a solution ?

**Comment by rawlingsj** [comment]
@theobolo yeah thats right, it's a little early for that but not too far away now that the [thirdpartyresource](http://kubernetes.io/v1.1/docs/design/extending-api.html) is available.

I'm not able to say if running origin on kubernetes is possible but in theory if you're able to do that then yes you should be able to run the CD pipeline.

**Comment by theobolo** [comment]
@rawlingsj Yep ok, for the moment i just use the Fabric8 console to manage easly my Kubernetes cluster and i've installed a fonctionnal Jenkins from that article : http://www.infoq.com/articles/scaling-docker-kubernetes-v1

I've tried to install Openshift on the Azure K8S cluster but it's not so easy :/ and i prefer to wait a little bit and keep following the Fabric8 developpement.

Should i keep to post issues here, related with my config or not ?

**Comment by rawlingsj** [comment]
@theobolo yeah sounds like a good plan.  Yeah feel free to carry on posting issues here related to fabric8 and we'll do our best to look at them sooner rather than later.  Ensuring there's a consistent experience across vanilla kubernetes and openshift is important.
